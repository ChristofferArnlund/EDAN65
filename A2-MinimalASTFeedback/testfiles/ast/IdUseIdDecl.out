Program
  List
    Func
      IdDecl
        Name ID="a"
        Opt
      List
      StmtBlock
        List
          IdDecl
            Name ID="x"
            Opt
          IdDecl
            Name ID="y"
            Opt
          Assign
            IdUse
              Name ID="x"
            Numeral NUMERAL="5"
          Assign
            IdUse
              Name ID="y"
            IdUse
              Name ID="x"
