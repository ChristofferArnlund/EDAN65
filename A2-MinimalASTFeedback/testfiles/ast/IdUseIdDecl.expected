Program
  List
    Func
      IdDecl ID="a"
        Opt
      List
      StmtBlock
        List
          IdDecl ID="x"
            Opt
          IdDecl ID="y"
            Opt
          Assign
            IdUse ID="x"
            Numeral NUMERAL="5"
          Assign
            IdUse ID="y"
            IdUse ID="x"
